src/components/VerifyUserDetailsModal.jsx

```diff
@@ -213,7 +213,7 @@ const VerifyUserDetailsModal = () => {
   }
 
   return userId && (!loggedInEmail || !loggedInContactNumber) ? (
-    <div className={elementStyles.overlay}>
+    <div data-testid="verify-user-modal" className={elementStyles.overlay}>
```

Comment:
nit: i think we are already using `data-cy` for test-related ids - could we do it here too? if we are removing the `visit` overwrite, we should also remove this

Reply:
Good point - I've changed it to `data-cy` in https://github.com/isomerpages/isomercms-frontend/pull/913/commits/48e7038677c6b0197796f1ec3839fb3a6c75da5f, though I'm open to removing it if we're removing the visit override

Reply:
Removed in https://github.com/isomerpages/isomercms-frontend/pull/913/commits/fc2f482e470441554b09768aa7fa3e807e0abf58!